pkg/interop/convert/convert.go

```diff
@@ -20,3 +22,133 @@ func ToString(v any) string {
 func ToBool(v any) bool {
 	return v.(bool)
 }
+
+// Uint8ToBytes converts a uint8 to a fixed-length slice.
+func Uint8ToBytes(n uint8) []byte {
+	return []byte{n & 0xff}
+}
+
+// BytesToUint8 converts data to a uint8.
+func BytesToUint8(data []byte) uint8 {
+	if len(data) != 1 {
+		panic("expected data with length 1")
+	}
+	return data[0]
+}
+
+// Uint16ToBytesBE converts a uint16 to a fixed-length slice.
+func Uint16ToBytesBE(n uint16) []byte {
+	var res = make([]byte, 2)
+	copy(res, ToBytes(n))
+	neogointernal.Opcode1NoReturn("REVERSEITEMS", res)
+	return res
+}
+
+// BytesBEToUint16 converts data in big-endian format to a uint16.
+func BytesBEToUint16(data []byte) uint16 {
+	if len(data) != 2 {
+		panic("expected data with length 2")
+	}
+	src := make([]byte, 2)
+	copy(src, data)
+	neogointernal.Opcode1NoReturn("REVERSEITEMS", src)
+	// Add 0 to reach a 2's complement form.
+	src = append(src, 0)
```

Comment:
Inline the comment. And this one is very abstract, it won't be correct if we ever change `bigint.FromBytes` behaviour. So we do it here to preserve compatibility with `bigint.FromBytes`, that's the only thing that developer should know if we ever modify this piece of code in future.
```suggestion
	src = append(src, 0) // preserve compatibility with bigint.FromBytes.
```

Reply:
Ditto below.